packages/bpk-component-barchart/src/BpkBarchartBar.js

```diff
@@ -40,7 +42,22 @@ const handleKeyboardEvent = callback => event => {
 
 const borderRadius = remToPx(borderRadiusXs);
 
-const BpkBarchartBar = props => {
+type Props = {
+  height: number,
+  label: string,
+  width: number,
+  x: number,
+  y: number,
+  className: ?string,
+  onClick: ?(?any) => mixed,
```

Comment:
```suggestion
  onClick: ?() => mixed,
```

I'm 99% sure this is the same thing, so might as well lose the `?any` part.

Reply:
Sadly not as the function in use within the component expects a param to be passed to the function so this is saying 

onClick could be null - but when its passed a param is required and its current type is any as the consumer could pass in whatever they want as the param